product/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+
+class Category(models.Model):
+    name                = models.CharField(max_length = 45)
+    description         = models.TextField
+    ingredients_image   = models.CharField(max_length = 1000)
+    recommended_image   = models.CharField(max_length = 1000)
+    great_for_image     = models.CharField(max_length = 1000)
+    ingredients         = models.CharField(max_length = 300)
+    recommended         = models.CharField(max_length = 300)
+    great_for           = models.CharField(max_length = 300)
+
+    class meta :
+        db_tables = 'categories'
+
+class DetailImage(models.Model):
+    category        = models.ForeignKey('Category', on_delete = models.CASCADE)
+    detail_image    = models.CharField(max_length = 1000)
```

Comment:
```suggestion
    image_url    = models.URLField(max_length = 2000)
```
테이블이 설명하고 있으니 앞에 detail은 생략해도 괜찮을것 같습니다.